src/main/java/org/opentripplanner/routing/algorithm/filterchain/package.md

```diff
@@ -0,0 +1,34 @@
+# ItineraryListFilterChain
+
+ItineraryListFilterChain is a mechanism for post-processing itinerary results. It contains a list of filters, which 
+contain the business logic of filtering. the filter chain is also responsible for creating routing errors, if all 
```

Comment:
Add that filtering is not just removal, but decorating/sorting/adding itineraries